environment/src/main/java/com/sequenceiq/environment/environment/service/EnvironmentService.java

```diff
@@ -207,9 +208,25 @@ public CreateAWSEnvironmentRequest getCreateAWSEnvironmentForCli(EnvironmentDto
         return environmentRepository.findByResourceCrnAndAccountIdAndArchivedIsFalse(resourceCrn, accountId);
     }
 
+    /**
+     * The CIDR could not be edited so the first step we clear the the existed CIDR every time.
+     * We are assuming the security access has the proper security group ids and we don't check it again.
```

Comment:
why?

Reply:
Because I check the security access one step before it. I don't want to check here again, so if anyone wants to use this function, calculate with that, I clear the cidr and set anything that they set to SecurityAccess without validation.